src/main/scala-sbt-0.13/compat.scala

```diff
@@ -10,6 +10,15 @@ object compat {
   implicit class ModuleIdOps(val _m: ModuleID) extends AnyVal {
     def withName(n: String): ModuleID = _m copy (name = n)
   }
+
+  implicit class RichProcessBuilder(pb: ProcessBuilder) {
+    def lineStream_!(logger: ProcessLogger) = pb.lines_!(logger)
+    def lineStream_! : Stream[String]       = pb.lines_!
+  }
+
+  implicit def promoteProcessLogger(logger: sbt.Logger): ProcessLogger =
+    ProcessLogger(msg => logger.info(msg), msg => logger.error(msg))
```

Comment:
Hmm, why is this required? There is already an implicit lift from Logger to ProcessLogger in sbt 0.13:

https://github.com/sbt/sbt/blob/v0.13.16/util/log/src/main/scala/sbt/Logger.scala#L58

Reply:
sbt.Logger is not available in sbt 1.0.

Reply:
In sbt 1 sbt.Logger still exists. It's defined at sbt.util.Logger (https://github.com/sbt/util/blob/v1.0.2/internal/util-logging/src/main/scala/sbt/util/Logger.scala#L19) and then aliased as sbt.Logger (https://github.com/sbt/sbt/blob/v1.0.2/sbt/src/main/scala/Import.scala#L81-L82)

Reply:
indeed, it's not needed you are right.

Reply:
however, you need to  `import sbt.Logger._'. I think it would make sense to make it available here as well. This way, you only need to 'import sbt.compat._' and it solves several issues with one import.

Reply:
Are you sure? Unless I'm missing something by being defined on the companion object it's in implicit scope, for both sbt 0.13 and sbt 1, so it's ok?

Btw I'm happy to find a way to include this is need by, I'm just not convinced of the need.